unstructured/documents/elements.py

```diff
@@ -377,9 +380,9 @@ def __init__(
         element_id: Union[str, NoID] = NoID(),
         coordinates: Optional[List[float]] = None,
         coordinate_system: Optional[CoordinateSystem] = None,
-        metadata: ElementMetadata = ElementMetadata(),
+        metadata: Optional[ElementMetadata] = None,
     ):
-        super().__init__(text="<PAGE BREAK>")
```

Comment:
@qued , I thought this was a clean solution here, I'm usually a fan of explicitly stating which kwarg is being passed in in case the position ever changes: 
```python
    def __init__(self, **kwargs):
        super().__init__(text="<PAGE BREAK>")
```

Reply:
I think this would break in the case someone using it passed positional arguments.

Reply:
Can I update it to `def __init__(self, *args, **kwargs):` then since none of them are actually used.

Reply:
Agree on passing `text=...` though.

Reply:
Per our offline conversation, it's fine if we go back to the args/kwargs pattern if we aren't passing on the arguments.